templates/issue.html

```diff
@@ -283,6 +283,14 @@
 			</div>
 		</div>
 		<div>
+			<div class="participants"> 
+				<div class="number-participants">{{$.NumberOfCommented}} participant</div>
+				<div class="participants-photo">
+				{{range $.CommentedUsers}}
+					<div class="user-photo"><img class="user-photo" src="{{.}}"></div>
```

Comment:
- img is size too big.
- Rounded corners.

Reply:
done